core/cap-0034.md

```diff
@@ -254,30 +494,40 @@ then rejected, as described above in the "Design Rationale".
 
 ## Test Cases
```

Comment:
This section still mentions implementation details of stellar-core.

Reply:
I've removed all test and method names from that section.